packages/gestalt/src/utils/positioningUtils.js

```diff
@@ -176,9 +176,16 @@ export function getPopoverDir({
 
   let popoverDir;
 
-  if (idealDirection && spaces[DIR_INDEX_MAP[idealDirection]] > 0) {
+  if (
+    idealDirection &&
+    idealDirection !== 'forceDown' &&
+    spaces[DIR_INDEX_MAP[idealDirection]] > 0
+  ) {
     // user pref
     popoverDir = idealDirection;
+  } else if (idealDirection && idealDirection === 'forceDown') {
+    // user pref
+    popoverDir = 'down';
```

Comment:
if this is the only logic, then can you just do if(idealDirection === 'forceDown') can you just return at the top. No need for all the other logic right?

Reply:
this was a refactor, done but kinda OOScope

Reply:
DONE